### PR TITLE
[tf2tfliteV2-conversion-test] Upgrade TF to 2.12.1

### PR DESCRIPTION
**Pull request by seanshpark**
iThis will upgrade test environment TF to 2.12.1.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #11255 , from draft  #11278

File: compiler/tf2tfliteV2-conversion-test/CMakeLists.txt [hunk 1]
```diff
@@ -76,12 +76,7 @@ list(APPEND TEST_DEPS "${TEST_RUNNER}")
 
 get_target_property(ARTIFACTS_BIN_PATH testDataGenerator BINARY_DIR)
 
-# TODO Run both 2.8.0 and 2.10.1 test for jammy
-if(ONE_UBUNTU_CODENAME_JAMMY)
```
Comment (seanshpark) [inline compiler/tf2tfliteV2-conversion-test/CMakeLists.txt:80]: Now all versions can use TF 2.12.1.

**Review by jinevening — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
> fatal error: error writing to /tmp/cc0NUvpY.s: No space left on device

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-jammy-debug

**Comment by seanshpark** [comment]
> [2023-08-21T09:14:25.886Z] ERROR: externals/tflite-1.13.1/CMakeFiles/tensorflowlite-1.13.1.dir/build.make:1766: recipe for target 'externals/tflite-1.13.1/CMakeFiles/tensorflowlite-1.13.1.dir/opt/jenkins_agent/workspace/nnfw/master/pr-nncc-debug/externals/TENSORFLOW-1.13.1/tensorflow/lite/kernels/mul.cc.o' failed
> [2023-08-21T09:14:25.886Z] make[2]: *** [externals/tflite-1.13.1/CMakeFiles/tensorflowlite-1.13.1.dir/opt/jenkins_agent/workspace/nnfw/master/pr-nncc-debug/externals/TENSORFLOW-1.13.1/tensorflow/lite/kernels/mul.cc.o] Error 1
> [2023-08-21T09:14:27.243Z] make[2]: write error: stdout

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-debug

**Pull request merged**